appveyor.yml

```diff
@@ -25,16 +25,19 @@ environment:
       CC: clang
       CXX: clang++
       SQLITE_ORM_CXX_STANDARD: "-DSQLITE_ORM_ENABLE_CXX_14=ON"
+      cmake_build_parallel: --parallel
 
     - job_name: gcc, C++14
       appveyor_build_worker_image: Ubuntu
       CC: gcc
       CXX: g++
       SQLITE_ORM_CXX_STANDARD: "-DSQLITE_ORM_ENABLE_CXX_14=ON"
+      # gcc was stuck with a parallel build
+      cmake_build_parallel: ""
 
     # Representative for C++14
-    - job_name: Visual Studio 2022, x64, C++14
-      appveyor_build_worker_image: Visual Studio 2022
+    - job_name: Visual Studio 2019, x64, C++14
+      appveyor_build_worker_image: Visual Studio 2019
```

Comment:
@fnc12 There's a little stupid problem with unit tests on Windows.

The Visual Studio 2019 build worker image only has CMake 3.16.
With this PR I introduced a dependency on CMake 3.24, because of the `FIND_PACKAGE_ARGS` feature (which prefers an already installed Catch2 library over building it from source, which takes quite some build time).

Contrary to Visual C++ 2019, Visual C++ 2017 has still broken variadic template pack expansion, which we treat as a compiler quirk in sqlite_orm.
But of course it would be useful to have the representative C++14 test on Windows run with VC++ 2017.